package.json

```diff
@@ -302,7 +303,8 @@
     "postinstall": "node scripts/patchReactVirtualized.js",
     "predev": "npm run build-types",
     "release": "release-it",
-    "prepare": "is-ci || husky install"
+    "prepare": "is-ci || husky install",
+    "dep-status": "depcruise --config .dependency-cruiser.js src"
```

Comment:
**question**: should `.dependency-cruiser.js` be committed into repo?

Reply:
Tried to use it via npx but didn't work - seems like there is no executable published